roofit/batchcompute/inc/RooSpan.h

```diff
@@ -48,7 +48,7 @@ class RooSpan {
   { }
 
 
-  /// Conversion constructor from <T> to <const T>
+  /// Conversion constructor from `<T>` to `<const_T>`
```

Comment:
```suggestion
  /// Conversion constructor from `RooSpan<NON_CONST_T>` to `RooSpan<T>`, where `T` is a `const` type.
```
I would suggest a notation that is consistent with the actual code.